operators/ipfs-cluster/0.0.1/metadata/annotations.yaml

```diff
@@ -0,0 +1,14 @@
+annotations:
+  # Core bundle annotations.
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: ipfs-operator
```

Comment:
@RobotSail plase fix package name to `ipfs-cluster` to `ipfs-operator` or rename `operators/ipfs-cluster` to `operators/ipfs-operator`. Package name has to be same as operator directory name. Since in csv there is name `ipfs-cluster.v0.0.1`i would recomend you to just rename operator directory to `ipfs-operator`

Reply:
Gotcha, thanks!